doc/sampler.rst

```diff
@@ -105,23 +105,62 @@ How to setup a Redis based distributed cluster
 ----------------------------------------------
 
 
-Step 1: Start a Redis server without password authentication
+Step 1: Reconfigure the redis.conf file
 ~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
 
-Start one some machine, which is reachable by the machine running the pyABC
-main application and by the workers, a Redis server, disabling password
-authentication:
+It is advised to run Redis server in protected mode. Authenticated
+communication will allow only for authenticated access to communicated with
+the server, and reject all unauthorised access. To run Redis server with
+authentication required you need first to modify the redis.conf file.
+
+The redis.conf is the file that contains Redis configuration. Usually,
+it can be found on `/etc/redis/`. to allow safe and secure communication,
+redis.conf file should be reconfigure as follow:
```

Comment:
I think the safest thing for the user will be to copy that file to his home directory, and then only edit that file, and pass it to ``redis-server ``.

Reply:
This would also make the part about "How to start a redis server without password" simpler: No need to undo anything, just don't specify that conf file in the redis-server command ...